Dockerfile

```diff
@@ -23,7 +23,11 @@ ENV SRC_PATH="/bubuku"
 ADD ./bubuku "${SRC_PATH}/bubuku"
 ADD ./requirements.txt "${SRC_PATH}/"
 ADD ./setup.py "${SRC_PATH}/"
-RUN cd "${SRC_PATH}" && pip3 install --no-cache-dir -r "requirements.txt" && python3 setup.py develop
+RUN cd "${SRC_PATH}" && \
```

Comment:
From old Dockerfile in bubuku-appliance, there is also:
 `chmod 777 /run_bubuku_daemon.sh`, do you think we should add this too?

Reply:
I'm not sure this is needed. I want to be sure it's needed before adding it.

Reply:
Got it, thanks!